libraries/core_mqtt.cmake

```diff
@@ -11,9 +11,21 @@ afr_set_lib_metadata(IS_VISIBLE "true")
 # Include MQTT library's source and header path variables.
 include("${CMAKE_CURRENT_LIST_DIR}/coreMQTT/mqttFilePaths.cmake")
 
+# Create a list of all header files in the coreMQTT library.
+# The list of header files will be added to metadata required
+# for the Online Configuration Wizard.
+set(MQTT_HEADER_FILES "")
+foreach(mqtt_public_include_dir ${MQTT_INCLUDE_PUBLIC_DIRS})
+    file(GLOB mqtt_public_include_header_files
+              LIST_DIRECTORIES false
+              ${mqtt_public_include_dir}/* )
+    list(APPEND MQTT_HEADER_FILES ${mqtt_public_include_header_files})
+endforeach()
+
 afr_module_sources(
     ${AFR_CURRENT_MODULE}
     PRIVATE
+        ${MQTT_HEADER_FILES}
```

Comment:
Can't we directly use ${MQTT_INCLUDE_PUBLIC_DIRS} in here?

Reply:
We need to have the entire file path like `coreMQTT/source/include/core_mqtt.h` to add the header file in the metadata. The `{MQTT_INCLUDE_PUBLIC_DIRS}` only contains the directory path to the header files.

Reply:
Ok. Thanks.